Observer/Customer.php

```diff
@@ -71,6 +71,10 @@ public function execute(Observer $observer)
             $customer = $this->getCustomerFromEvent($observer);
             
             if ($customer) {
+                if (!$this->helper->isEnabled($customer->getStoreId())) {
```

Comment:
combine this and the check on line 73 in one that just returns early

Reply:
Combined on line 73, no need to return, code will be skipped if one of conditions is not true.

Reply:
This was silly, now it's combined in one statement that requires $customer object and enabled module status to continue with the code execution.